src/components/CourseDetail/CourseDetail.js

```diff
@@ -1,9 +1,46 @@
 import React, { useState, useEffect } from 'react';
 import { useLocation } from "react-router-dom";
 import axios from 'axios';
+import { useSelector } from 'react-redux';
 
 import ExperienceCard from '../ExperienceCard/ExperienceCard';
 
+/* helper method to get an icon given its backend-configured name */
```

Comment:
Can be made more maintainable and readable using an object rather than a switch. Create an object where each cases' name is mapped to the name of the icon found on the back end. Do a `if` check to see if the object comes back `undefined`. If the object is not undefined return the Icon and the name associated with it. Otherwise return the default case.